Comment _MM_SET_FLUSH_ZERO_MODE for now.

gcc-4.7.3 with -mtune=generic uses STMXCSR/VSTMXCSR instructions which are
only available in Sandy Bridge CPUs, thus causes a SIGILL on AMD CPUs.

--- src/common/darktable.c.orig	2014-05-08 11:42:15.844554271 +0200
+++ src/common/darktable.c	2014-05-08 11:43:00.724288778 +0200
@@ -363,7 +363,7 @@ int dt_init(int argc, char *argv[], cons
 #endif
 
   // make everything go a lot faster.
-  _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
+  //_MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);
 #if !defined __APPLE__ && !defined __WIN32__
   _dt_sigsegv_old_handler = signal(SIGSEGV,&_dt_sigsegv_handler);
 #endif
